### PR TITLE
Basic talk extension promo box

### PR DESCRIPTION
**Pull request by tackley**
Screenshot 2022-02-10 at 16 23 34

### CODE REVIEW

**Comment by mrose17** [comment]
@tackley - what do you think about dismissing the notification when someone clicks on "Install Browser Extension"? (Otherwise, when someone clicks on it, and then goes back to that tab, the notification is still there!)

Also, when I tested it, clicking on "Install Browser Extension" did not set the flag in `localStorage`... is that on purpose?

**Comment by tackley** [comment]
@mrose17 good points both. will make it both dismiss and set the flag when you click the Install buttons, it does neither at the moment which was an oversight rather than deliberate.

File: src/css/welcome.css [hunk 1]
```diff
@@ -523,7 +523,69 @@ button {
   }
 }
 
+.extension-promo {
+  background: pink;
```
Comment (tackley) [inline src/css/welcome.css:527]: remove this debug!

File: src/css/welcome.css [hunk 2]
```diff
@@ -523,7 +523,69 @@ button {
...
+  position: fixed;
+  bottom: 16px;
+  right: 16px;
+
+  background: linear-gradient(
+      139.76deg,
+      rgba(255, 255, 255, 0.5) 71.26%,
+      rgba(76, 84, 210, 0.0001) 115.29%
+    ),
+    #ffffff;
```
Comment (tackley) [inline src/css/welcome.css:538]: dark mode styling required

Comment (tackley) [reply src/css/welcome.css:536]: done in https://github.com/brave/brave-talk/pull/174/commits/c6be8e121fa9f4f35ad23a3714d26029f1ca6b96

File: src/css/welcome.css (hunk 1 above)
Comment (tackley) [reply src/css/welcome.css]: done in https://github.com/brave/brave-talk/pull/174/commits/c6be8e121fa9f4f35ad23a3714d26029f1ca6b96

**Comment by mrose17** [comment]
My issues are addressed, thank you! Now awaiting feedback from design.

**Comment by bradleyrichter** [comment]
Text is ready:

Title: Now you can schedule ahead!
Body: Add Brave Talk links directly to an invite, with the Google Calendar extension.
Button: Install Calendar extension

**Comment by tackley** [comment]
Updated copy:

Screenshot 2022-02-14 at 12 37 01

**Comment by tackley** [comment]
Screenshot 2022-02-14 at 19 05 33

**Review by mrose17 — APPROVED** [review]
Both my comments and Brad's comment have been addressed. Approved! Please promote to staging for QA.

**Pull request merged**